python/pysrc/kaskada/windows.py

```diff
@@ -16,10 +16,15 @@ class Window(object):
 
 @dataclass(frozen=True)
 class Since(Window):
-    """Window since the last time a predicate was true.
+    """A series of non-overlapping windows that reset each time a predicate evaluates to true.
 
     Aggregations will contain all values starting from the last time the predicate
     evaluated to true (inclusive).
+
+    Aggregations will produce a new value for each input value and additionally when
+    the current window closes.
```

Comment:
Not sure about the term "closes". Maybe "and when the predicate evaluates to true causing the window to reset (including all of the results up to that point)".

Reply:
What do you dislike about "close"? 

Window "open" and window "close" sound like good terms to adopt. (I like it better than "start/end"). 

"Reset" feels more like an implementation strategy than a user-facing explanation, sometimes. The "reset" or "clear" window language feels more like "restarting the window", but without implying that the window is also shifting forward when it's reset. "Open" and "Close" imply to a user that the existing window is "closed", meaning no more values can be added to the window in this time frame, and then a new one in a new time boundary is "opened".

Reply:
Note: I know "reset" is used elsewhere. Petitioning to change the language everywhere to use "open" and "close" instead of "reset"